neural_lam/models/base_graph_model.py

```diff
@@ -115,7 +112,6 @@ def predict_step(
             (
                 prev_state,
                 prev_prev_state,
-                batch_static_features,
```

Comment:
Note that the batch-static features are now put as the first feature dimension in forcing. Earlier they were stacked right on top of forcing in this tensor. This results in no change to how `grid_features` looks like for a sample. Importantly, this means that models trained before this PR can be loaded and works without any problems.